packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -832,6 +832,10 @@ class VirtualListBasic extends Component {
 		}
 	}
 
+	updateScrollPositionTarget (x, y) {
```

Comment:
If you are going to define this function, why not change the code block from `setScrollPosition` and `scrollToPosition` to this function?